src/components/AccordionList/AccordionList.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Accordion } from "react-bootstrap";
+import AccordionItem from "./AccordionItem";
+
+export default function AccordionList({ id="", list }) {
+	return (
+		<Accordion
+			alwaysOpen
+			id={id}>
+			{list.map((e, i) => (
+				<AccordionItem
+					eventKey={i}
+					key={`${id}-${i}`}
```

Comment:
Be careful with using indexes in component keys. If the order of the list ever changes, you could have strange bugs: https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/